DB/api/db.py

```diff
@@ -66,6 +75,80 @@ def query(parsed_url):
 		return format_data(header, results, json)
 
 
+def generate_total_energy(device_id, start_time, end_time, channel):
+	"""
+	Returns total energy for a particular "channel" (device in table data_raw),
+	over a specified time period
+
+	:param device_id: The serial number of the device in question
+	:param start_time: The start of the time range for which to query for data
+	:param end_time: The end of the time range for which to query for data
+	:param channel: channel filter
+	"""
+
+	# Initialize parameter list and WHERE clause
+	params = [device_id]
+	query = "SELECT CAST(data as DECIMAL) / 36e5 FROM " + TABLE + " as raw WHERE serial = %s AND type = 'P'"
```

Comment:
This is the accurate way to get energy from the power data. @aadabi and i verified it last night